src/templates/base.html

```diff
@@ -13,13 +13,43 @@
     {% block head %}
       <title>{% block title %}The Web Almanac{% endblock %}</title>
       <script async src="https://www.googletagmanager.com/gtag/js?id=UA-22381566-3" nonce="{{ csp_nonce() }}"></script>
+      <script async src="/static/js/web-vitals.js" ></script>
       <script nonce="{{ csp_nonce() }}">
           window.dataLayer = window.dataLayer || [];
           function gtag() {
               dataLayer.push(arguments);
           }
           gtag('js', new Date());
           gtag('config', 'UA-22381566-3');
+
+          function sendWebVitalsGAEvents(metric) {
+            let event_value = 0;
+            switch (metric.name) {
+              case 'CLS':
+                // For CLS event value is first multiplied by 1000 for greater precision.
+                event_value = Math.round(metric.delta * 1000);
+                break;
+              default:
+                event_value = metric.delta;
```

Comment:
Actually shouldn't the non-CLS metrics be rounded too?
```suggestion
                eventValue = Math.round(metric.name === 'CLS' ? metric.delta * 1000 : metric.delta);
```
Should we just use [syntax like they suggest](https://github.com/GoogleChrome/web-vitals#using-gtagjs) which would replace a lot of this code and potentially make it easier to include any future changes by comparing to their suggested code?

Reply:
Yeah, you're right. Event value takes integer: https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#ev